paymentintent/client.go

```diff
@@ -0,0 +1,131 @@
+// Package paymentintent provides API functions related to payment intents.
+//
+// For more details, see: https://stripe.com/docs/api/go#payment_intents.
+package paymentintent
+
+import (
+	"net/http"
+
+	stripe "github.com/stripe/stripe-go"
+	"github.com/stripe/stripe-go/form"
+)
+
+// Client is used to invoke /payment_intents APIs.
+type Client struct {
+	B   stripe.Backend
+	Key string
+}
+
+// New creates a payment_intent.
```

Comment:
Nit, but would you mind if we used the more human-readable "payment intent" for these comments?

Reply:
damn search and replace. Nice catch and fixed!